src/main/java/com/curateme/claco/member/domain/entity/Gender.java

```diff
@@ -0,0 +1,23 @@
+package com.curateme.claco.member.domain.entity;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+/**
+ * @author      : 이 건
+ * @date        : 2024.10.18
+ * @author devkeon(devkeon123@gmail.com)
+ * ===========================================================
+ * DATE               AUTHOR        NOTE
+ * -----------------------------------------------------------
+ * 2024.10.18   	   이 건        최초 생성
+ */
+@Getter
+@AllArgsConstructor
+public enum Gender {
+
+	MALE("MALE"), FEMALE("FEMALE");
```

Comment:
아래와 같이 트레일링 컴마를 붙이면 추후 코드변경이 발생했을때, 변경되는 라인이 적어서 코드리뷰하기 용이합니다.

```suggestion
	MALE("MALE"), 
	FEMALE("FEMALE"),
	;
```